.github/workflows/testing.yml

```diff
@@ -7,16 +7,19 @@ on:
 jobs:
   install-dependencies:
     runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        deno-version: [1.37.0]
```

Comment:
The supported latest deno version by deno-vercel is v1.36.1.
https://github.com/vercel-community/deno/releases

Reply:
The `vercel.json` already do it :)

Do i really need change the Runner ?

Reply:
changed

Reply:
OK